setup.py

```diff
@@ -33,13 +33,15 @@ def find_packages(path=["."], prefix=""):
             continue
         else:
             req = req.split(" ")[0]
+            # Work around normalized name of github3.py distribution
+            req = req.replace("github3-py", "github3.py")
```

Comment:
Huh. Figured setuptools/wheel would do the right thing here.

Reply:
Yeah -- I think there's probably a bug somewhere in there, but I hit my timebox for locating it and decided to do this instead.